src/Components/CookieConsent.js

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import * as Scrivito from "scrivito";
+import CookieConsent from "react-cookie-consent";
+
+/**
+ * @return {null}
+ */
```

Comment:
Please remove the comments, since they are not accurate and don't provide much value.